stl_text/datamodule/doc_classification.py

```diff
@@ -9,6 +9,30 @@
 from stl_text.ops.transforms import LabelTransform
 from torch.nn.utils.rnn import pad_sequence
 from stl_text.ops.samplers import PoolBatchSampler
+from iopath.common.file_io import PathManager
+
+from torchtext.experimental.transforms import (
+    sentencepiece_tokenizer,
+    PRETRAINED_SP_MODEL,
+)
+from torchtext.experimental.vocab import (
+    build_vocab_from_text_file,
+    Vocab,
+)
+
+from torchtext._torchtext import Vocab as VocabPybind 
+
+from torchtext.utils import download_from_url
+import torch.nn as nn
+
+class TextToIndices(nn.Module):
```

Comment:
This is the same transform as [TextSequentialTransforms](https://github.com/pytorch/text/blob/master/torchtext/experimental/transforms.py#L170). Also you can use a lambda func.